torch/jit/annotations.py

```diff
@@ -149,15 +149,15 @@ def get_type_line(source):
     # https://www.python.org/dev/peps/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code
     return_line = None
     parameter_type_lines = []
-    for line_num, line in reversed(type_lines):
+    for line_num, line in type_lines:
         if '# type: (...) -> ' in line:
             return_line = (line_num, line)
+            break
         elif type_comment in line:
-            if return_line is None:
-                raise RuntimeError("Return type line '# type: (...) -> ...' not found on multiline "
-                                   "type annotation\n(See PEP 484 https://www.python.org/dev/peps/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code)")  # noqa
-            if line_num < return_line[0]:
-                parameter_type_lines.insert(0, line)
+            parameter_type_lines.append(line)
+    else:
```

Comment:
Can you re-arrange this to get rid of the `for..else`? It's concise but too many of us PyTorch devs will have to google it to know what it means

Reply:
Yes, I will update the pull request.